services/user/Supervisor/ui/src/plugin/pluginHost.ts

```diff
@@ -22,6 +25,15 @@ interface HttpResponse {
     body: string;
 }
 
+const SupervisorPromptPayload = z.object({
```

Comment:
`zSupervisorPromptPayload` and `.min(1)` instead of `.nonempty()`. According to the docs, the latter is for validating arrays; it seems to be deprecated for strings.